docs/topics/gradle/gradle-configure.md

```diff
@@ -0,0 +1,743 @@
+[//]: # (title: Configure your existing project)
```

Comment:
"Configure a Gradle project (project with Gradle?)" it fits better: this article is not about configuring any existing or new projects, but _Gradle_ projects.